templates/partials/menu.html

```diff
@@ -1,45 +1,38 @@
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="datacatalog.html">Data Catalog</a>
-                    </li>
-                   <li class="nav-item">
-                        <a class="p-2 menu-link" href="ide.html">IDE</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="remote-processing.html">Remote Processing</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="tutorials.html">Tutorials</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="about.html">About</a>
-                    </li>
-                </ul>
-            </div>
-            <div class="col-1">
-                <a class="social-media-icon" href="https://www.linkedin.com/company/marbleclimate" target="_blank"><i class="fa-brands fa-linkedin fa-lg"></i></a>
-                <a class="social-media-icon" href="https://twitter.com/marble_climate" target="_blank"><i class="fa-brands fa-square-x-twitter fa-lg"></i></a>
-                <a class="social-media-icon" href="https://mastodon.social/@marbleclimate" target="_blank" rel="me"><i class="fa-brands fa-mastodon fa-lg"></i></a>
-                <a class="social-media-icon" href="https://www.instagram.com/marbleclimate" target="_blank"><i class="fa-brands fa-square-instagram fa-lg"></i></a>
-                <!-- TODO: uncomment discourse link below when we have a discourse instance to link to -->
-                <!-- <a class="social-media-icon" href="" target="blank"><i class="fa-brands fa-discourse fa-lg"></i></a> -->
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <div class="col-1">
+                    <img id="menu-logo" src="images/logos/marble.png" class="img-fluid ms-4 d-none d-lg-block menu-logo" />
+                </div>
+                <div class="col-10">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="p-2 menu-link" aria-current="page" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="p-2 menu-link" href="index.html#technology">Technology</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="p-2 menu-link" href="nodes.html">Nodes</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="p-2 menu-link" href="tutorials.html">Tutorials</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="p-2 menu-link" href="index.html">Community</a>
```

Comment:
This can go to discourse.marbleclimate.com now

Reply:
Made the changes to the link.  Included "https://" in front for now so it can be accessed from localhost.